factory/runType/sovereignRunTypeComponents.go

```diff
@@ -157,6 +159,9 @@ func (rcf *sovereignRunTypeComponentsFactory) Create() (*runTypeComponents, erro
 		return nil, fmt.Errorf("sovereignRunTypeComponentsFactory - NewSovereignAccountCreator failed: %w", err)
 	}
 
+	expiryTime := time.Second * time.Duration(rcf.cfg.OutgoingSubscribedEvents.TimeToWaitForUnconfirmedOutGoingOperationInSeconds)
+	outGoingOperationsPoolCreator := sovereignFactory.NewOutGoingOperationPool(expiryTime)
```

Comment:
This is not a creator, it is the outgoing op pool
```suggestion
	outGoingOperationsPool:= sovereignFactory.NewOutGoingOperationPool(expiryTime)
```